bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/AbstractConfiguration.java

```diff
@@ -1098,7 +1098,7 @@ public int getAllocatorPoolingConcurrency() {
      *            the concurrency level to use for the allocator pool
      * @return configuration object.
      */
-    public T setAllocatorPoolingConcurrenncy(int concurrency) {
+    public T setAllocatorPoolingConcurrency(int concurrency) {
```

Comment:
This is a public API change and it will break users.

In order to fix this property we have to:
- add the new method
- mark the existing method as Deprecated
- wait for a minor (4.18) version release
-  send a new a patch that removes the method

Reply:
@eolivelli Thanks for your review and suggestions, I have updated and I will follow the removal for old method.